src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -60,11 +60,8 @@ public void createFeedbackQuestion(FeedbackQuestionAttributes fqa)
 
         String feedbackSessionName = fqa.feedbackSessionName;
         String courseId = fqa.courseId;
-        List<FeedbackQuestionAttributes> questions = null;
-
-        try {
-            questions = getFeedbackQuestionsForSession(feedbackSessionName, courseId);
-        } catch (EntityDoesNotExistException e) {
+        List<FeedbackQuestionAttributes> questions = getFeedbackQuestionsForSession(feedbackSessionName, courseId);
+        if (questions.isEmpty()) {
```

Comment:
Empty question list does not mean that the session doesn't exist.

Should use `fsLogic.getFeedbackSession` to check if session exist.

Reply:
I have fixed it. I see build failed because several test failing.

Reply:
Made the change.